dehydrated

```diff
@@ -576,6 +577,29 @@ init_system() {
     grep -q newOrder <<< "${CA_DIRECTORY}" && API=2 || API=1
   fi
 
+  # Get available certificate profiles from CA
+  CA_PROFILES="$(printf "%s" "${CA_DIRECTORY}" | get_json_dict_value -p '"meta","profiles"')" || CA_PROFILES=""
+
+  # Validate requested ACME profile if specified
+  if [[ -n "${ACME_PROFILE}" ]]; then
+    profiles_array=$(echo "${CA_PROFILES}" | get_json_array_values)
```

Comment:
The `meta.profiles` from `GET /directory` is a dict rather than array. Are you sure that `get_json_array_values` works properly against a JSON dict?